### ISSUE TITLE
Fill in sponsors info, switch main menu link to sponsors

### ISSUE THREAD

**Issue by phette23** [code4lib/2018.code4lib.org#71]
(assigned to Ben Wallberg, will have to look up his GitHub…)

Copy _data/examples/sponsors.yml and fill in the accurate sponsor information including names, logos (optional), links, and captions. Activate the sponsors menu link by turning `call-for-sponsors` at the very bottom of _data/conf.yml to `false`.

See also #4 — we want to display prominent sponsors on the home page once this issue is complete.

**Comment by ghost** [comment]
@phette23 Ben Wallberg GitHub username is wallberg-umd

**Comment by ghost** [comment]
I'm finding a few bugs in the Sponsors display as I work on this.  Should I bundle the fixes in with the sponsors info or would you prefer a separate issue and PR?

**Comment by phette23** [comment]
It's be great if you could just bundle the fixes in, thanks. And sorry about misspelling your family name 😬 

I've asked for you to be added to the code4lib github organization and once you're in I can assign you to this issue.
